cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -0,0 +1,151 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable
+import org.jenkinsci.plugins.workflow.cps.Snippetizer
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor
+import org.jenkinsci.plugins.workflow.structs.DescribableHelper
+
+import javax.servlet.RequestDispatcher
+
+Snippetizer snippetizer = my;
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+
+h1(_("Steps"))
+for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
```

Comment:
So the reason I did it in Groovy is that, well, it was a lot easier to do it here in the view than to pass it to the view in some form and handle all the rendering logic separately in the view. I could certainly be wrong and there may well be a less awkward approach, but, well, this worked. =)